### DISCUSSION TITLE
Blog building error Error in file(file, ifelse(append, "a", "w")).... SOMETIMES?!

### DISCUSSION THREAD

**Discussion by MayaGans** [quarto-dev/quarto-cli#1809] [Q&A]
Okay this is really weird and a horrible question because I don't have any reprex because it's super hard to reproduce (which is also part of the problem!) but when building a Quarto blog using the Render button I *sometimes* get this error:

```
Preparing to preview
[ 1/11] posts/recursion-again-and-again-and-again/index.qmd  
Error in file(file, ifelse(append, "a", "w")) : 
  cannot open the connection
Calls: .main -> execute -> write -> cat -> file
In addition: Warning message:
In file(file, ifelse(append, "a", "w")) :
  cannot open file 'index.qmd': Operation not permitted
Execution halted
```

as for `[ 1/11] posts/recursion-again-and-again-and-again/index.qmd ` I built this just as a document outside the blog and it was fine?!  

The way I've been resolving it is to restart my R session, take all the posts outside of my post folder, and then add them back in and then it works?!?! ...And then I'll make some changes and see the error again??

Does anyone have any ideas on how to even debug this? Any help appreciated!

**Comment by cscheid** [comment]
Hi, sorry you're running into trouble. Is this in a github repo somewhere so we can try to trigger the bug ourselves?

**Reply by MayaGans** [reply]
omg thank you! https://github.com/MayaGans/mywebsite <- I removed a bunch of posts because trouble shooting but there's one in there `[exploring-networks-through-concerts](https://github.com/MayaGans/mywebsite/tree/main/posts/exploring-networks-through-concerts)` that works and `[why-would-i-settimeout0](https://github.com/MayaGans/mywebsite/tree/main/posts/why-would-i-settimeout0)` is causing me the error 😭

**Comment by jjallaire** [comment]
Hi Maya, thanks for the repro. I cloned it locally and am also not able to reproduce the error message. Are you seeing this every time now or is it still somewhat non-deterministic?

**Reply by MayaGans** [reply]
OMG it's working today 😆  [I promise I did the whole restart the R session thing, and my entire computer for that matter....] Maybe I just needed to sacrifice something to the Quarto gods??? So odd! We can close this and I'll ping you both if it happens again! Sorry to bother you and thank you so much for looking into it!

**Reply by jjallaire** [reply]
Glad to hear!!
